README.md

```diff
@@ -338,27 +338,34 @@ gateway:
       type: RuntimeDefault
     runAsUser: 1003
 loki:
+  overrideConfiguration:
+    storage_config:
+      aws:
+        bucketnames: <BUCKET NAME>
+        endpoint: <S3 ENDPOINT URL>
+        access_key_id: <ACCESS KEY>
+        secret_access_key: <SECRET>
+        s3forcepathstyle: true
+    schema_config:
+      configs:
+      - from: "2024-01-01"
```

Comment:
Does this hardoded value is normal?

Reply:
short answer: yes it is ok
long answer: The date indicates that any log newer than this date should use the storage type listed in the following line (tsdb) which is the recommended for the current stable version of loki. We could have added another element in the list with different dates if we were deploying over a previously used loki version, in which case we would need to add a date range and a different storage type